__tests__/app.test.js

```diff
@@ -35,4 +33,33 @@ describe('GET api/topics', () => {
             })
         })
     })
+})
+
+describe('GET api/articles', () => {
+    test('200 - responds with array of articles objects with following properties: author, title, article_id, topic, created_at, votes, comment_count', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body: {articles}}) => {
+            expect(articles).toHaveLength(12);
+            articles.forEach((article) => {
+                expect(article).toEqual(expect.objectContaining({
+
+                    article_id: expect.any(Number),
+                    comment_count: expect.any(String),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
```

Comment:
body is the only property articles are not meant to have - other than that, test looks solid 👍